yt/yt/ytlib/api/native/connection.cpp

```diff
@@ -1104,10 +1126,8 @@ class TConnection
             }));
     }
 
-    //! Returns a pair consisting of the client for the Query Tracker cluster and the path to the Query Tracker root in Cypress.
-    std::pair<IClientPtr, TString> GetQueryTrackerStage(const TString& stage) override
-    {
-        auto findStage = [&stage, this] (const TString& cluster) -> std::pair<IClientPtr, TString> {
+    std::pair<IClientPtr, TQueryTrackerStageConfigPtr> FindQueryTrackerStage(const TString& stage) {
```

Comment:
[Opening braces of classes and functions are placed on the next line.](https://github.com/ytsaurus/ytsaurus/blob/main/yt/styleguide/cpp.md#braces)

Can we move it back?